snooty/test_parser.py

```diff
@@ -200,7 +203,7 @@ def test_literalinclude() -> None:
 """,
     )
     page.finish(diagnostics)
-    assert len(diagnostics) == 1
+    assert len(diagnostics) == 1 and isinstance(diagnostics[0], CannotOpenFile)
```

Comment:
I want to confirm that this should be a CannotOpenFile error, which is what I got when I ran the tests and not a InvalidLiteralInclude like the above asserts?